types/branca/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for branca 0.3
+// Project: https://github.com/tuupola/branca-js
+// Definitions by: Brett Cocking <https://github.com/BrettIRL>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export = branca;
+
+declare class Branca {
+    constructor(key: string);
+
+    encode(message: string, timestamp?: number): string;
+    decode(token: string, ttl?: number): Buffer | Error;
```

Comment:
Unless I'm missing something, this method can throw, but never _returns_ an error.

Reply:
You are 100% correct. No idea where I got that from. Will get sorted.